lte/gateway/python/scripts/config_stateless_agw.py

```diff
@@ -0,0 +1,188 @@
+        return return_codes.STATELESS
+    elif num_stateful == len(service_config_names):
+        print("Check returning", return_codes.STATEFUL)
+        return return_codes.STATEFUL
+
+    print("Check returning", return_codes.CORRUPT)
+    return return_codes.CORRUPT
+
+
+def check_stateless_agw():
+    sys.exit(_check_stateless_services().value)
+
+
+def _clear_redis_state():
+    if os.getuid() != 0:
+        print("Need to run as root to clear Redis state.")
+        sys.exit(return_codes.INVALID)
+    subprocess.call("service magma@* stop".split())
+    subprocess.call("service magma@redis start".split())
+    subprocess.call("redis-cli -p 6380 FLUSHALL".split())
+    subprocess.call("service magma@redis stop".split())
+
+
+def _start_magmad():
+    if os.getuid() != 0:
+        print("Need to run as root to start magmad.")
+        sys.exit(return_codes.INVALID)
+    subprocess.call("service magma@magmad start".split())
+
+
+def _restart_sctpd():
+    if os.getuid() != 0:
+        print("Need to run as root to restart sctpd.")
+        sys.exit(return_codes.INVALID)
+    print("Restarting sctpd")
+    subprocess.call("service sctpd restart".split())
+
+
+def enable_stateless_agw():
+    if _check_stateless_services() == return_codes.STATELESS:
+        print("Nothing to enable, AGW is stateless")
+        sys.exit(return_codes.STATELESS.value)
+    for service, config in service_config_names:
+        cfg = load_override_config(service)
+        if cfg is None:
+            cfg = {}
+        if service == "pipelined":
+            cfg[config] = False
+        else:
+            cfg[config] = True
+
+        save_override_config(service, cfg)
+
+    # restart Sctpd so that eNB connections are reset and local state cleared
+    _restart_sctpd()
+    sys.exit(_check_stateless_services().value)
+
+
+def disable_stateless_agw():
+    if _check_stateless_services() == return_codes.STATEFUL:
+        print("Nothing to disable, AGW is stateful")
+        sys.exit(return_codes.STATEFUL.value)
+    for service, config in service_config_names:
+        cfg = load_override_config(service)
+        if cfg is None:
+            cfg = {}
+
+        # remove the stateless override
+        cfg.pop(config, None)
+
+        save_override_config(service, cfg)
+
+    # restart Sctpd so that eNB connections are reset and local state cleared
+    _restart_sctpd()
+    sys.exit(_check_stateless_services().value)
+
+
+def sctpd_pre_start():
+    if _check_stateless_services() == return_codes.STATEFUL:
+        # switching from stateless to stateful
+        print("AGW is stateful, nothing to be done")
+    else:
+        _clear_redis_state()
+    sys.exit(0)
+
+
+def sctpd_post_start():
+    _start_magmad()
+    time.sleep(15)  # sleep for a bit to ensure OVS and Magma services are up
+    sys.exit(0)
+
+
+def clear_redis_and_restart():
+    _clear_redis_state()
+    _start_magmad()
+    sys.exit(0)
+
+
+function_map = {
```

Comment:
define this as uppercase as it will be a constant dict, probably `STATELESS_FUNC_DICT` might be a more descriptive name for this

Reply:
Fixed in latest commit https://github.com/magma/magma/pull/2914/commits/b721dd8bd07990f698426b117ee611a9252cb5db